nucypher/cli/actions/select.py

```diff
@@ -67,7 +67,8 @@ def select_stake(staker: Staker,
         raise click.Abort
 
     # Interactive Selection
-    paint_stakes(staker=staker, emitter=emitter, stakes=stakes)
+    paint_unlocked = stakes_status is Stake.Status.INACTIVE
```

Comment:
Since this is a utility function, are we causing issues if UNLOCKED is ever passed to this function, since we are only specifically checking for INACTIVE here (unsure if there is ever such a need, but from a logic perspective)? i.e. should this instead be:

```suggestion
    paint_unlocked = stakes_status.is_child(Stake.Status.UNLOCKED)
```

```python
In [8]: Stake.Status.UNLOCKED.is_child(Stake.Status.UNLOCKED)
Out[8]: True

In [9]: Stake.Status.INACTIVE.is_child(Stake.Status.UNLOCKED)
Out[9]: True
```

Reply:
I think it's more like bug in `paint_stakes` because if `stakes` is not None then just print without checking status